### ISSUE TITLE
Update headings on dashboard-details screen

### ISSUE THREAD

**Issue by felixarntz** [google/site-kit-wp#460]
## Feature Description

Wording and layout in the dashboard details needs to be updated. See ACs for details.

These are new requirements based on #138, which will be closed.

---------------

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance criteria

* All sections should receive titles, and some titles should receive updated wording:
    * Single Page Stats --> Detailed Page Stats
    * The box with impressions, clicks, etc should have title "Search Funnel" and description "How your page appeared in Search results and how many visitors you got from Search."
    * The box that has "Acquisition Sources" should have title "All Traffic" and description "How people found your page." instead.
    * The box that has "Top search queries for your page" should have title "Top Queries" and description "What people searched for to find your page."
* Titles (and the new descriptions) should appear similarly to the dashboard, above each box. It should use the same styling.
* The date range dropdown should remain as is - just one at the top.

## Implementation Brief

* 

## Changelog entry

*

**Comment by felixarntz** [comment]
@aaemnnosttv Apologies, apparently one idea from the ACs didn't become clear: The boxes in `dashboard-details` screen that already have a single heading (inside the box) shouldn't maintain these headings. For example, the "Acquisition Sources" header with the Analytics link should completely vanish. Both the "Acquisition Sources" and the "Top search queries for your page" boxes only have that single heading, so the header should be removed from these, and we'd instead rely solely on the new headings provided above the box (which you've already covered).

In other words, the IB needs to be expanded to cover changes to the props passed to the `Layout` component for these widgets.

**Comment by felixarntz** [comment]
@aaemnnosttv Sorry, I think there's still a minor inaccuracy, see

> the header should be removed from these

In other words, we shouldn't pass the `header` prop or any related prop any longer.

Other than that, IB ✅

**Comment by aaemnnosttv** [comment]
Thanks @felixarntz, I've updated the IB to mention those as well. Were there any other related props you were referring to that should also be removed?

**Comment by adamsilverstein** [comment]
This looks good in my testing, moving to "Acceptance".

Confirmed the headers appear as specified, I also verified the date selector works as expected.

![image](https://user-images.githubusercontent.com/2676022/64568182-5a0caf00-d317-11e9-9cd6-95887e8f86de.png)
